BSP/Simulator/Data/twotest.json

```diff
@@ -1,6 +1,6 @@
 [
     {
-        "time": 10,
+        "time": 1,
```

Comment:
why is the time changed to 1 in this test case? Also, could you put a screenshot of the simulator output in the comments?

Reply:
<img width="372" alt="image" src="https://user-images.githubusercontent.com/24827956/204643333-b6cec741-d4cd-4aad-83ea-990c95972dac.png">
i was trying different times srry